hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -329,7 +329,7 @@ public Record loadRecordOrNull(Data key, boolean backup, Address callerAddress)
         if (value != null) {
             record = createRecord(value, DEFAULT_TTL, DEFAULT_MAX_IDLE, getNow());
             storage.put(key, record);
-            mutationObserver.onPutRecord(key, record);
+            mutationObserver.onLoadRecord(key, record);
```

Comment:
Prior to `3.11` this method didn't publish events hence we write `LOADED` here unconditionally. See line 335 below.